src/commonControls/multiValueListControl/MultiValueListControl.ts

```diff
@@ -1438,6 +1501,42 @@ export class MultiValueListControl extends Control implements InteractionModelCo
         generator.ensureSlotValueIDsAreDefined(this.id, 'target', this.props.interactionModel.targets);
     }
 
+    private addBuiltInIntents(generator: ControlInteractionModelGenerator) {
+        this.addRemoveByOrderIntent(generator);
+    }
+
+    private addRemoveByOrderIntent(generator: ControlInteractionModelGenerator) {
+        generator.addIntent({
+            name: 'GeneralMultiValueListControl_RemoveByOrderIntent',
```

Comment:
Instead of creating a custom intent here, maybe we can extend `OrdinalControlIntent` and create another Intent module called `RemoveOrdinalsControlIntent` with unpackRemoveOrdinalsControlIntent methods and add in updateInteractionModel as 
```js
 generator.addControlIntent(new RemoveOrdinalsControlIntent(), imData);
 ```